docs/learn/learn-comparisons.md

```diff
@@ -1,63 +1,212 @@
+| **Decentralization**         | Centralization risks for sequencers and zk proof computation.                                                                                                                                                                                                  | Sequencer centralization risks; fewer validators can lead to potential censorship.                                                                                           | Collator decentralization is critical to ensure parachains remain trustless and independent.                                                                                                                                                                                                                                                        |
+| **Interoperability**         | Limited interoperability, often restricted to compatible L1s.                                                                                                                                                                                                  | Limited interoperability, often confined to the parent blockchain ecosystem. See the [comparison about interoperability](#interoperability-comparison) for more information. | Native interoperability through [XCM](./learn-xcm.md), allowing seamless communication between parachains having different logic. [Trustless bridges](./learn-bridges.md) can connect Polkadot to other blockchains.                                                                                                                                |
+| **State Transition Logic**   | General-purpose but constrained by zk-circuit implementation complexity.                                                                                                                                                                                       | Can only support EVM state transitions with assumptions of optimistic validity since they have to be checked in the EVM environment<sup>1</sup>.                             | Each parachain can define its unique state transition function (STF), compiled to Wasm and validated per Parachain Protocol rules.                                                                                                                                                                                                                  |
```

Comment:
Not true, optimistic rollups' opcodes aren't natively checked by the L1. Instead, the individual opcodes of the L2 are interpreted by the EVM.
One could equally implement the interpretation logic of other VMs aside from the EVM.